flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/TestTriggerable.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.api.operators;
+
+import org.assertj.core.api.Assertions;
+
+import java.util.function.Consumer;
+
+/** Triggerable for test. */
+class TestTriggerable<K, N> implements Triggerable<K, N> {
+
+    private final Consumer<InternalTimer<K, N>> eventTimeHandler;
+    private final Consumer<InternalTimer<K, N>> processingTimeHandler;
+
+    public static <K, N> TestTriggerable<K, N> eventTimeTrigger(
+            Consumer<InternalTimer<K, N>> eventTimeHandler) {
+        return new TestTriggerable<>(
+                eventTimeHandler,
+                timer -> Assertions.fail("We did not expect processing timer to be triggered."));
+    }
+
+    public static <K, N> TestTriggerable<K, N> processingTimeTrigger(
```

Comment:
This method can be removed.

Reply:
I updated the test `BatchExecutionInternalTimeServiceTest` so that the method is used.